build.gradle

```diff
@@ -82,7 +82,7 @@ dependencies {
     def jacksonVersion = "2.15.1"
     def jacksonDataBindVersion = "2.14.1"
     def junitVersion  = "4.13.1"
-    def log4jVersion = "2.20.0"
+    def log4jVersion = "2.21.0"
```

Comment:
Can we take dependency from core so as to avoid future version upgrades.
https://github.com/opensearch-project/performance-analyzer/blob/5ab3345ed8f9b076d5095cfa7248767a1fa35580/build.gradle#L272

Reply:
Hey @praveensameneni , yeah we should use dependency from core for all the above version numbers. But since this is a emergency change to unblock the 2.11 release today, let's make the change and revisit it later.